plugins-compat-tester/src/test/resources/jenkins-bom.xml

```diff
@@ -129,7 +129,7 @@
     <structs-plugin.version>1.20</structs-plugin.version>
     <maven-localizer-plugin.version>1.24</maven-localizer-plugin.version>
     <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
-    <workflow-api-plugin.version>2.37</workflow-api-plugin.version>
+    <workflow-api-plugin.version>1200.v8005c684b_a_c6</workflow-api-plugin.version>
```

Comment:
?

Reply:
need to test both plugins contains a real tag and a sha1 in their poms.
anyway the unit tests are not testing anything... :(